backend/src/test/java/com/senior/project/backend/portfolio/PortfolioHandlerTest.java

```diff
@@ -0,0 +1,57 @@
+package com.senior.project.backend.portfolio;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import com.senior.project.backend.domain.User;
+import com.senior.project.backend.domain.YearLevel;
+import com.senior.project.backend.portfolio.dto.EditEducationDTO;
+import com.senior.project.backend.util.Endpoints;
+
+import reactor.core.publisher.Mono;
+
+@ExtendWith(MockitoExtension.class)
+public class PortfolioHandlerTest {
+    private WebTestClient webTestClient;
+
+    @InjectMocks
+    private PortfolioHandler portfolioHandler;
+
+    @Mock
+    private PortfolioService portfolioService;
+
+    @BeforeEach
+    public void setup() {
+        webTestClient = WebTestClient.bindToRouterFunction(new PortfolioRouter().portfolioRoutes(portfolioHandler)).build();
+    }
+
+    @Test
+    public void testSaveEducationSuccess() {
+        when(portfolioService.saveEducation(any(EditEducationDTO.class))).thenReturn(Mono.just(new User())); // Mock the service call
```

Comment:
Should the param be Mono.nonempty since we are checking it against a:
.expectStatus().isNoContent(); 204 No Content
This change and the other two fixes the CI but do verify if the intent of the tests hold:
when(portfolioService.saveEducation(any(EditEducationDTO.class))).thenReturn(Mono.empty());